src/test/java/seedu/address/commons/util/StringUtilTest.java

```diff
@@ -109,8 +109,11 @@ public void containsWordIgnoreCase_validInputs_correctResult() {
         assertFalse(StringUtil.containsWordIgnoreCase("    ", "123"));
 
         // Matches a partial word only
-        assertFalse(StringUtil.containsWordIgnoreCase("aaa bbb ccc", "bb")); // Sentence word bigger than query word
-        assertFalse(StringUtil.containsWordIgnoreCase("aaa bbb ccc", "bbbb")); // Query word bigger than sentence word
+
+        // Query word is substring of sentence word
+        assertTrue(StringUtil.containsWordIgnoreCase("aaa bbb ccc", "bb"));
+        // Sentence word is substring of sentence word
```

Comment:
The comment for this test seems incorrect?

Reply:
Thanks for catching it! I have made the edits accordingly